Add a blog post on office hours (#3925)
tenzir/tenzir: web/blog/office-hours/index.md

---
title: Introducing Office Hours
authors: [dominiklohmann]

> date: 2024-02-13

Author: Must not be merged before next Tuesday.

tags: [discord, office-hours]
comments: true
---

Did you ever want to get a sneak peek behind the scenes at Tenzir? Now you can!

Introducing: **Office Hours**

📅 Every Tuesday at 8 AM EST / 11 AM EST / 5 PM CET / 9.30 PM IST  

> 👉 Sign up here: [Discord](https://discord.gg/JGS2N2Dwf6?event=1202543209662906393)

Reviewer: Did you set the link to never expire?

Author: Yep.

<!-- truncate -->

## What are Office Hours?

Starting immediately, our team will host a dedicated Office Hours event every
week on our Discord server. This is an open-door event where every one of you is
welcome, no matter what your level of experience or expertise.

## What should you expect from this?

Office Hours is an exclusive space where you bring up your questions, thoughts,
suggestions, or just pop in to say hi. From giving insights about our operations
over discussing the latest updates to providing exclusive previews of upcoming
features, we will ensure that each session is loaded up with useful information
for you. It's also an excellent opportunity for you to get a sneak peek behind
the scenes, know us better, and see how we make the magic happen!

## The topics we'll cover

We encourage you to bring any kind of questions to the table. Whether it's about
our work, our team, our future plans, or whatever else comes to your mind. While
we want these hours to be helpful, we also want to make it a space where we can
share, learn, and grow together as a community.

## How you can participate

Getting involved is simple:

> 1. If you're not already a part of our Discord server, click this
>    [link](discord.tenzir.com) to join.

Reviewer: ```suggestion
1. If you're not already a part of our Discord server,
   [join us](https://tenzir.com/discord)!
```

2. We'll announce the  hours before they are starting every Tuesday at 8 AM EST
   / 11 AM EST / 5 PM CET / 9.30 PM IST.
3. During the event, head over to the dedicated Office Hours voice channel on
   our Discord server to join.

> And voila! You are all set.

Reviewer: ```suggestion
And voilà! You are all set.
```

Author: I did change it to `Et voilà` now.
